jubilant/_juju.py

```diff
@@ -905,6 +1061,40 @@ def trust(
 
         self.cli(*args)
 
+    def update_secret(
+        self,
+        name_or_uri: str | SecretURI,
+        /,
+        content: Mapping[str, str],
+        *,
+        info: str | None = None,
+        name: str | None = None,
+        auto_prune: bool = False,
+    ) -> None:
+        """Update the content of a secret.
+
+        Args:
+            name_or_uri: The name or the secret uri for the secret.
+            content: Key-value pairs that represent the secret content, for example
+                ``{'password': 'hunter2'}``.
+            info: Optional description for the secret.
+            name: Optional new name for the secret.
```

Comment:
The type annotations indicate these are optional, so I'd do:

```suggestion
            info: New description for the secret.
            name: New name for the secret.
```

Reply:
We have docs that start with Optional on `run` and `deploy` too. Should I remove those too?

Reply:
updated the `run` and `deploy` docstring too.